ocs/ceph.py

```diff
@@ -1033,8 +1035,7 @@ def connect(self):
 
         stdin, stdout, stderr = self.rssh().exec_command("dmesg")
         self.rssh_transport().set_keepalive(15)
-        changepwd = 'echo ' + "'" + self.username + ":" + self.password + "'" \
-                    + "|" + "chpasswd"
+        changepwd = 'echo ' + "'" + self.username + ":" + self.password + "'" + "|" + "chpasswd"
```

Comment:
This is long line 96 chars, we don't force to line length 79 but we should follow as we agreed I think.